Sprint-3/revise/implement/password-validator.test.js

```diff
@@ -14,3 +14,9 @@ To be valid, a password must:
 
 You must breakdown this problem in order to solve it. Find one test case first and get that working
 */
+const isValidPass = require('./password-validator');
+
+test("Is prime number", function(){
+  expect(isValidPass('Calohh35#')).toBe(true);
+  expect(isValidPass('alohh35#')).toBe(false);
+});
```

Comment:
These tests does not check if the function can correctly return `false` for each specific type of invalid passwords.
For example, the test only check if the function can return false when a password does not contain any uppercase letter, but the test does not check if the function can return false when a password is missing a lowercase.

Can you include more test cases to see if the function can return false for each type of invalid passwords? They could possibly help you **detect a bug** in your code.

Reply:
I missed out the digit checks in the regular expressions.